.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-    "stage": 3,
+    "stage": 2,
```

Comment:
Just want to confirm this doesn't have wide-ranging implications outside of this PR?